src/ztgamemgr.rs

```diff
@@ -0,0 +1,180 @@
+// ztgamemgr module has functions to interact with the live zoo stats such as cash, num animals, species, guests, etc.
+
+use crate::add_to_command_register;
+use crate::debug_dll::{get_from_memory, get_string_from_memory};
+
+use tracing::info;
+use std::collections::HashMap;
+use std::fmt;
+use std::fmt::Display;
+use num_enum::FromPrimitive;
+
+const GLOBAL_ZTGAMEMGR_ADDRESS: u32 = 0x00638048;
+
+// ZTGameMgr struct
+#[derive(Debug)]
+#[repr(C)]
+struct ZTGameMgr {
+    instance: u32, // 0x0
+    get_date: String, // 0x1194 (8 bytes)
+    get_cash: f32, // 0x0C
+    set_cash: f32, // 0x0C
+    add_cash: f32, // 0x0C
+    sub_cash: f32, // 0x0C
```

Comment:
Shouldn't this all just be a single member called `cash`?

Reply:
`add_cash`, `sub_cash`, `set_cash` are all in the game as their own functions. right now they (and other pointers) lazily point to their respective resource until I have time to flesh them out as either rewrites, proper function pointers, or raw object structure reimplementation with byte padding for alignment. Functionally they're the same for now.